Dockerfile.migrations

```diff
@@ -0,0 +1,10 @@
+FROM registry.devshift.net/bayesian/bayesian-api
+MAINTAINER Slavek Kabrda <slavek@redhat.com>
```

Comment:
Any reasons why this is here (database migrations)? Also MAINTAINER is [deprecated](https://docs.docker.com/engine/reference/builder/).